setup.cfg

```diff
@@ -1,11 +1,14 @@
-##########################
-# Setup.py Configuration #
-##########################
+###################################################
+# Setup.py Configuration                          #
+# Adapted from https://github.com/pybel/pybel.git #
+###################################################
+
 [metadata]
 name = coronawhy_vt
 version = 0.0.1-dev
 description = CoronaWhy Vaccines and Therapeutics Task Force
-long_description = file: README.rst
```

Comment:
would we be okay with switching this over to RST as well?

Reply:
Sounds good, I haven't worked with RST files, but from a bit of googling, this definitely seems appropriate